src/classes/services/HttpService.cls

```diff
@@ -4,10 +4,10 @@
 
 public class HttpService {
 
-    public static String makeHttpRequest(String callout, String urlData, String method, Map<String, String> apiKeyMap){
+    public static HTTPResponse makeHttpRequest(String callout, String urlData, String method, Map<String, String> apiKeyMap){
         HttpRequest req = new HttpRequest();
         req.setMethod(method);
-        req.setEndpoint(callout+'/'+urlData);
```

Comment:
MINOR
You can replace 'callout' and 'urlData' with single param, like 'enpoint', there is no need to keep both, you just require more params to the method.